lib/compiler/src/compile.erl

```diff
@@ -1148,6 +1173,27 @@ foldl_transform([T|Ts], Code0, St) ->
     end;
 foldl_transform([], Code, St) -> {ok,Code,St}.
 
+%%% It is possible--although unlikely--that parse transforms add
+%%% columns to the abstract code, why this function is called for
+%%% every parse transform.
+maybe_strip_columns(Code, T) ->
+    case erlang:function_exported(T, parse_transform_info, 0) of
```

Comment:
I'm afraid this check might not be successful since in many cases parse transforms would be lazily loaded, and `function_exported` returns `false` for unloaded modules